verticapy/sdk/vertica/udf/gen.py

```diff
@@ -74,6 +88,23 @@ def generate_lib_udf(
     -------
     udx_str, sql
         UDF py file, str needed to install the library.
+
+    Example
+    -------
+    .. ipython:: python
+
+        from verticapy.udf import generate_lib_udf
+        @suppress
+        import math
+
+        udx_str, udx_sql = generate_lib_udf([(math.exp, [float], float, {}, "python_exp"),
+                                             (math.isclose, [float, float], bool, {"abs_tol": float}, "python_isclose"),],
+                                              library_name = "python_math",
+                                              file_path = "",
+                                              create_file = False)
+        print(udx_str)
```

Comment:
is it possible to separate it into two blocks so the user can clearly see the result?

Ex:
VerticaPy will generate the UDX Python code:
```
 print(udx_str)
```
And the SQL needed to install the function:
```
print("\\n".join(udx_sql))
```